test/installer.js

```diff
@@ -113,4 +114,30 @@ describe('module', function () {
 
     it('generates a `.rpm` package', () => assertNonASARRpmExists(outputDir))
   })
+
+  describeInstaller(
+    'with an app with installation scripts as paths',
```

Comment:
This implies that you can specify installation scripts as not-paths (inline?) somehow...

Reply:
Yes, it does. Inline is not the preferred way, as scripts can be lengthy, that's why I did not document this option. Currently, users need to pass a `scripts` object with the paths, but they can also pass `options.pre`, `options.post`, etc. containing the script's body as a string and that would also work. This is a side effect on how templates work with lodash. We need to send the `options` object to the template to generate the specs file and that object has to contain the contents of the scripts already, so people could just add the contents of the scripts directly to the `options` object